src/test/java/me/xmrvizzy/skyblocker/chat/filters/AutopetFilterTest.java

```diff
@@ -0,0 +1,21 @@
+package me.xmrvizzy.skyblocker.chat.filters;
+
+import me.xmrvizzy.skyblocker.chat.ChatPatternListenerTest;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertLinesMatch;
+
+import java.util.List;
+
+class AutopetFilterTest extends ChatPatternListenerTest<AutopetFilter> {
+    public AutopetFilterTest() {
+        super(new AutopetFilter());
+    }
+
+    @Test
+    void testAutopet() {
+        List<String> expected = List.of(listener.pattern.toString());
+        List<String> actual = List.of("§cAutopet §eequipped your §7[Lvl 85] §6Tiger§e! §a§lVIEW RULE");
+
+        assertLinesMatch(expected, actual);
```

Comment:
The reason why I made ChatPatternListenerClass is to reduce code duplication. Use it.

If you ever need to check if string matches a pattern in other context, you can do it like this:
```java
boolean stringMatchesRegex = pattern.matcher(string).matches();
```

Reply:
I forgot it, sorry about that. assertLinesMatch for old Test.